Common/Orders/StopLimitOrder.cs

```diff
@@ -64,7 +125,7 @@ public StopLimitOrder()
         /// <param name="stopPrice">Price the order should be filled at if a limit order</param>
         /// <param name="tag">User defined data tag for this order</param>
         public StopLimitOrder(string symbol, int quantity, decimal stopPrice, decimal limitPrice, DateTime time, string tag = "", SecurityType type = SecurityType.Base) :
-            base(symbol, quantity, OrderType.StopLimit, time, 0, tag, type)
+            base(symbol, quantity, OrderType.StopLimit, time, limitPrice, tag, type)
```

Comment:
Why did we update this constructor to pass the `limitPrice` in as the `price` to the base constructor?  This will update the order's Price property which should be the fill price. Limit prices are always the same as the fill price for limit orders.